### PR TITLE
Apply DenoError TooLarge

### PR DESCRIPTION
**Pull request by Tnarita0000**
I've modified error type in `js/buffer.ts` to DenoError  from Error, so added enum type to `msg.fbs`.

### CODE REVIEW

**Comment by kevinkassimo** [comment]
Please run `./tools/format.py` and `./tools/lint.py`

File: js/buffer.ts [hunk 1]
```diff
@@ -170,7 +171,10 @@ export class Buffer implements Reader, Writer {
       // don't spend all our time copying.
       copyBytes(this.buf, this.buf.subarray(this.off));
     } else if (c > MAX_SIZE - c - n) {
-      throw Error("ErrTooLarge"); // TODO DenoError(TooLarge)
+      throw new DenoError(
+        ErrorKind.TooLarge,
+        "The buffer can't grow because it becomes too large"
```
Comment (kitsonk) [inline js/buffer.ts]: This isn't the best English.  "The buffer cannot be grown beyond the maximum size." would be better.

File: js/buffer_test.ts [hunk 2]
```diff
@@ -130,6 +134,23 @@ test(async function bufferLargeByteWrites() {
...
+  init();
+  const tmp = new Uint8Array(72);
+  const growLen = Number.MAX_VALUE;
+  const xBytes = repeat("x", 0);
+  const buf = new Buffer(xBytes.buffer as ArrayBuffer);
+  const { nread, eof } = await buf.read(tmp);
+  try {
+    buf.grow(growLen);
+  } catch (e) {
+    assertEqual(
```
Comment (kitsonk) [inline js/buffer_test.ts]: Why don't you just assert on `e.Message` and validate that `e instanceof DenoError`.

File: js/buffer_test.ts [hunk 3]
```diff
@@ -65,6 +65,10 @@ async function empty(buf: Buffer, s: string, fub: Uint8Array): Promise<void> {
   check(buf, "");
 }
 
+function stringify(...args: any[]): string {
```
Comment (kitsonk) [inline js/buffer_test.ts]: I really don't think is necessary.

**Comment by Tnarita0000** [comment]
@kitsonk Thank you for your review.

I've modified.

**Pull request closed without merging**